spec.html

```diff
@@ -27763,7 +27764,8 @@ <h1>Number.prototype.toExponential ( _fractionDigits_ )</h1>
               1. [id="step-number-proto-toexponential-intermediate-values"] Let _e_, _n_, and _f_ be integers such that _f_ &ge; 0, 10<sup>_f_</sup> &le; _n_ &lt; 10<sup>_f_ + 1</sup>, the Number value for ℝ(_n_) &times; 10<sub>ℝ</sub><sup>ℝ(_e_) - ℝ(_f_)</sup> is _x_, and _f_ is as small as possible. Note that the decimal representation of _n_ has _f_ + 1<sub>ℝ</sub> digits, _n_ is not divisible by 10, and the least significant digit of _n_ is not necessarily uniquely determined by these criteria.
             1. Let _m_ be the String value consisting of the digits of the decimal representation of _n_ (in order, with no leading zeroes).
           1. If _f_ &ne; 0, then
-            1. Let _a_ be the first code unit of _m_, and let _b_ be the remaining _f_ code units of _m_.
+            1. Let _a_ be the first code unit of _m_.
+            1. Let _b_ be the last _f_ code units of _m_.
```

Comment:
I don't like this change because it loses the context that this is all the code units except the first.

Reply:
+1 to this, here and below where the operations are clearly segmenting code units into a head and tail, a single step seems quite natural compared to two steps.

Reply:
Suggested alternative wording?

Reply:
The existing wording seems better than the wording in this PR. I'd also be OK with

```
1. Let _a_ be the first code unit of _m_.
1. Let _b_ be the other _f_ code units of _m_.
```

Reply:
@bakkot @syg Addressed using Kevin's suggested phrasing.